src/App.js

```diff
@@ -1,13 +1,57 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  // this component is going to take care of state, and any change 
+  // handlers you need to work with your state
+
+  constructor(){
+      super();
+      this.state = {
+        list : [
+          {
+            task: 'Organize Garage',
+            id: 1528817077286,
+            completed: false
+          },
+          {
+            task: 'Bake Cookies',
+            id: 1528817084358,
+            completed: false
+          }
+        ]
+    }
+  }
+
+  toggleSelection = (itemId) => {
```

Comment:
No problem with the structure of this function but I want to point out that this function name could be more descriptive. For example, `toggleTodoCompletness` or `toggleTodoCompleteState` tells me a lot more about wha this function does than `toggleSelection` it will make your code easier to read and review in the long run, for employers and your future self.